.github/workflows/overall-tests.yml

```diff
@@ -37,14 +37,18 @@ jobs:
           pipfile-version: ${{ matrix.pipfile-version }}
 
       - name: Pytest
-        run: pipenv run pytest -m "not orchestrator_dispatcher and not standalone and not teste2e" --cov=taipy --cov-append --cov-report="xml:overall-coverage.xml" --cov-report term-missing tests
-
-      - name: Coverage
+        run: |
+          if [ "${{ matrix.os }}" == "ubuntu-latest" ] && ["${{ matrix.python-version }}" == "3.11"] && ["${{ matrix.pipfile-version }}" == "min"]; then
+            pipenv run pytest -m "not orchestrator_dispatcher and not standalone and not teste2e" --cov=taipy --cov-append --cov-report="xml:overall-coverage.xml" --cov-report term-missing tests
```

Comment:
I think you mentioned it but why don't we include the orchestrator and dispatcher tests for coverage?

Reply:
Because we don't run all the tests together because of the flaky tests.